src/Microsoft.Build.Tasks.Git/GitDataReader/GitRepository.cs

```diff
@@ -209,6 +209,15 @@ public static GitRepository OpenRepository(GitRepositoryLocation location, GitEn
         /// <returns>Null if the HEAD tip reference can't be resolved.</returns>
         internal string? ReadSubmoduleHeadCommitSha(string submoduleWorkingDirectoryFullPath)
         {
+            // submodules don't usually have their own .git directories but can when Arcade uberclone
```

Comment:
Remove any references to Arcade. This is not infrastructure code, this is a shipping product.

Reply:
Removed.